sorc/ncep_post.fd/WRFPOST.F

```diff
@@ -384,6 +384,12 @@ PROGRAM WRFPOST
 ! Read and set global value for surface physics scheme
           call ext_ncd_get_dom_ti_integer(DataHandle                      &
             ,'SF_SURFACE_PHYSICS',itmp,1,ioutcount, status )
+
+          ! J. Kenyon / 23 Aug 2024: GSL MPAS output erroneously
+          ! indicates SF_SURFACE_PHYSICS = 0 (should be 3), so overwrite
+          ! here:
+          if (modelname == 'RAPR' .and. submodelname == 'MPAS') itmp = 3 
+
```

Comment:
@jaymes-kenyon Please add a change log the beginning session of this file (line 40).